### ISSUE TITLE
Please add build instructions

### ISSUE THREAD

**Issue by GuillaumeGomez** [askama-rs/askama_playground#2]
I wanted to add some missing features but when I run `cargo run` (after running `git submodule update --init`) I get:

```
     Running `target/debug/play-rinja`
thread 'main' panicked at /home/imperio/.cargo/registry/src/index.crates.io-6f17d22bba15001f/js-sys-0.3.70/src/lib.rs:6035:9:
cannot call wasm-bindgen imported functions on non-wasm targets
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

So obviously, I'm missing something important. I suppose opening `index.html` should be enough once we build for wasm target. But I'm not sure so anyway, please add build/run instructions. :)

**Comment by Kijewski** [comment]
Oh, yeah, I should write a readme. :-/

In the meantime:

```sh
# git submodule update --remote --no-recommend-shallow
# cargo install trunk
trunk serve  
```

**Comment by GuillaumeGomez** [comment]
Say no more!

**Comment by Kijewski** [comment]
Fixed in #3 & #4. Thank you! :)
